lib/chef/resource/ifconfig.rb

```diff
@@ -21,16 +21,67 @@
 
 class Chef
   class Resource
-    # @example set a static ip on eth1
-    #   ifconfig '33.33.33.80' do
-    #     device 'eth1'
-    #   end
     class Ifconfig < Chef::Resource
       unified_mode true
 
       provides :ifconfig
 
       description "Use the **ifconfig** resource to manage interfaces on Unix and Linux systems."
+      examples <<~DOC
+      **Configure a network interface with a static IP**
+
+      ```ruby
+      ifconfig '33.33.33.80' do
+        device 'eth1'
+      end
+      ```
+
+      will create the following interface configuration:
+
+      ```
+      iface eth1 inet static
+        address 33.33.33.80
+      ```
+
+      **Specify a boot protocol**
```

Comment:
Is this example missing `bootproto`?

Reply:
That whole example seems bogus. I just nuked it.